ntbs-service/Pages/Notifications/Edit/TestResults.cshtml.cs

```diff
@@ -8,22 +10,37 @@ namespace ntbs_service.Pages.Notifications.Edit
 {
     public class TestResultsModel : NotificationEditModelBase
     {
+        private readonly ICultureAndResistanceService _cultureAndResistanceService;
+        private readonly ISpecimenService _specimenService;
+
         public TestResultsModel(
             INotificationService notificationService,
             IAuthorizationService authorizationService,
-            INotificationRepository notificationRepository)
+            INotificationRepository notificationRepository,
+            ICultureAndResistanceService cultureAndResistanceService,
+            ISpecimenService specimenService)
             : base(notificationService,
                    authorizationService,
                    notificationRepository)
         {
+            _cultureAndResistanceService = cultureAndResistanceService;
+            _specimenService = specimenService;
         }
 
         [BindProperty]
         public TestData TestData { get; set; }
 
+        [BindProperty]
+        public CultureAndResistance CultureAndResistance { get; set; }
+
+        [BindProperty]
+        public IEnumerable<Specimen> Specimens { get; set; }
```

Comment:
Do these need to be `BindProperties`? I don't think we're getting their values form the user's POSTs and GETs.